File: net/Link.Message.Client/content/complex/linear/Column.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Link.Message.Client.content.complex.linear
{

	/// <summary>
    /// Grid类型的线性消息里，对猎头的定义
	/// </summary>
	public class Column
	{
		public string Title { set; get; }
        [JsonConverter(typeof(StringEnumConverter))]
	    public Alignment Align { get; set; }
	    public int Width { set; get; }

	    public Column()
		{

		}

		public Column(string title, Alignment align, int width)
		{
			this.Title = title;
			this.Align = align;
			this.Width = width;
		}
	}
}

File: java/link-message-client/src/main/java/link/message/client/content/complex/linear/Column.java
package link.message.client.content.complex.linear;

/**
 * Grid类型的线性消息里，对猎头的定义
 * @author zhongt
 *
 */
public class Column {
	protected String    title;
	protected Alignment align;
	protected int       width;
	
	public Column() {
		
	}
	
	public Column(String title, Alignment align, int width) {
		this.title = title;
		this.align = align;
		this.width = width;
	}

	public int getWidth() {
		return width;
	}

	public void setWidth(int width) {
		this.width = width;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public Alignment getAlign() {
		return align;
	}

	public void setAlign(Alignment align) {
		this.align = align;
	}
}